comic_recommender.py

```
@@ -0,0 +1,15 @@
+from typing import List, Dict, Tuple, Optional
+
+Comic = Dict[str, any]
+
+
+class ComicRecommender:
+    def __init__(self, comics_db: List[Comic]):
+        self.comics = comics_db
+
+    def get_next_in_series(self, current) -> Optional[Comic]:
+        same_series = [
```

Flake8 (7.2.0)

[error] local variable 'same_series' is assigned to but never used

(F841)